src/panels/usesSummaryPanel/UseSummaryPanel.tsx

```diff
@@ -0,0 +1,65 @@
+import { Card, CardContent, CardHeader } from "@material-ui/core";
+import { Activities, Environments, IUse } from "entities/IUse";
+import React, { FunctionComponent, ReactNode } from "react";
+import { AviationSummary } from "./AviationSummary";
+import { LandSummary } from "./LandSummary";
+import { MaritimeSummary } from "./MaritimeSummary";
+
+interface UseSummaryPanelProps {
+  use: IUse;
+  titlePrefix: string;
+}
+
+export const UseSummaryPanel: FunctionComponent<UseSummaryPanelProps> = ({
+  use,
+  titlePrefix,
+}: UseSummaryPanelProps): JSX.Element => {
+  const title = getCardHeaderTitle(titlePrefix, use);
+  const useSummary = getUseSummary(use);
+
+  return (
+    <Card>
+      <CardContent>
+        <CardHeader title={title} />
+        {useSummary}
+      </CardContent>
+    </Card>
+  );
+};
+
+const getCardHeaderTitle = (titlePrefix: string, use: IUse): string => {
+  let title = `${titlePrefix} use: `;
+  title +=
+    use.activity === Activities.Other
+      ? use.otherActivity?.toUpperCase()
+      : use.activity.replace(/_/, " ");
+
+  if (use.purpose) {
+    title += ` (${use.purpose})`;
+  }
+
+  return title;
+};
```

Comment:
This looks _very similar_ if not identical to a composite of `formatUse` and `titleCase` in `mcaWritingStyleFormatter`.  Could those functions be re-used instead?

Reply:
Used `formatUse` function